GoogleChromeLabs/ps-analysis-tool
# Enhancements: Updates to topics EE

## Description
This PR makes UI/UX enhancements on topics EE alongside enabling interlinking to topics tax tree.


- [x] Hovering over the domain circle should show the info box.
- [x] Make domains or small circles clickable and highlight rows that have the same domains.
- [x] Use same circle UI for “Observed-by context domains:”
- [x] Internal link from Topics EE to Taxonomy tree.
- [x] Remove the topic from the top of the node.
- [x] Do not connect lines by default, connect as the user travels through nodes.
- [x] A website can have multiple topics, add such examples in the explorable explanation.

## Relevant Technical Choices



## Testing Instructions
- Open PSAT, navigate the topics panel, and open explorable explanations.
The animation should start. After a few visits, pause it, and now hover over any circle. The info box should appear and remain there.
- Now click on the small circles, it should highlight the table cells for that circle's value.
- In the table click on the topic name, it should open the taxonomy tree in an expanded view.
- Now come navigate back to EE, it should be the same as navigated earlier. 


## Additional Information:



## Screenshot/Screencast

https://github.com/user-attachments/assets/7934948c-6724-4d5a-bf9a-223f1900b11a




---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~



Fixes #856
